warehouse/admin/templates/admin/projects/detail.html

```diff
@@ -183,7 +183,7 @@ <h3 class="card-title">
 </div> <!-- .card #attributes -->
 
 {% if project.organization %}
-<div class="card card-secondary" id="maintainers">
+<div class="card card-secondary" id="organization">
```

Comment:
This sunck in from a copy&pasta another time, no impact since nobody uses the ID yet.